README.md

```diff
@@ -1,5 +1,117 @@
-# ts-npm-package-boilerplate
+# Export-Interfaces
+npm - [@mapcolonies/export-interfaces](https://www.npmjs.com/package/@map-colonies/export-interfaces)
+
+This package define and expose common interfaces & types for export tasks
+
+### Quickstart
+---
+1. install
+```
+npm install @mapcolonies/export-interfaces
+```
+
+### Data in Export-Interfaces
+export-interfaces exposes the 'Export Manager' interface with common functions to be implement
+```typescript
+export interface IExportManager {
+  exportTask: (data: ExportTaskRequest<TaskParameters>) => Promise<ExportTaskResponse>;
+  getGeometries: (jobId: string) => Promise<TaskGeometry[]>;
+  getEstimatedSize: (params?: GetEstimatedSizeParams) => Promise<number>;
+  getFileSize: (catalogRecordID: string, params: GetFileSizeParams) => Promise<number>;
+}
+```
+most of Export-Interfaces functions uses **[@turf/turf](https://www.npmjs.com/package/@turf/turf)** npm package for geographic data.
+
+---
+
+2. import & implement
+
+```typescript
+import { IExportManager } from '@map-colonies/export-interfaces';
+
+export class ExportManager implements IExportManager {
+    ...
+}
+```
+
+3. implements IExportManager functions
+
+- **exportTask** example:
+```typescript
+export class ExportManager implements IExportManager {
+  public constructor(@inject(SERVICES.LOGGER) private readonly logger: Logger) {}
+
+  public async exportTask(data: ExportTaskRequest<TaskParameters>): Promise<ExportTaskResponse> {
```

Comment:
change the name to something more meaningful as createExportTask

Reply:
tests, lint and prettier

Reply:
fixed